specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2024-09-01-preview/appconfiguration.json

```diff
@@ -1870,6 +1852,58 @@
         }
       }
     },
+    "SasAuthProperties": {
+      "type": "object",
+      "description": "The SAS authentication settings of the configuration store.",
+      "properties": {
+        "status": {
+          "description": "The status of the SAS token authentication. This property manages if SAS token authentication is enabled or disabled.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SasAuth",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Enabled",
+                "description": "SAS token authentication is enabled."
+              },
+              {
+                "value": "Disabled",
+                "description": "SAS token authentication is disabled."
+              }
+            ]
+          }
+        },
+        "info": {
+          "description": "The information of SAS token authentication.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SasTokenInfo"
+          }
+        }
+      }
+    },
+    "SasTokenInfo":{
+      "type": "object",
+      "description": "The information of specific SAS token kind.",
+      "properties":{
+        "kind": {
+          "description": "The kind of the SAS token.",
+          "$ref": "#/definitions/SasTokenKind"
+        },
+        "lastResetTime":{
```

Comment:
I'm leaning towards calling it `lastModifiedTime`. Its a well known concept and it covers the scenario where SAS keys have not been reset by users, but SAS keys were still modified due to enable/disable of SAS auth. What do you think?

Reply:
I was thinking SAS keys concept is not exposed to users so might be better to explicitly say "reset", But I am ok with lastModifiedTime